react-web/src/components/atoms/typography/label/index.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames'
+
+import { FontSize } from 'components/enums/font-size'
+import { Status } from 'components/enums/status'
```

Comment:
In this case, you can add the enum on a single line.
`import { FontSize, Status } from 'components/enums'`